packages/trpc/server/routers/loggedInViewer/deleteCredential.handler.ts

```diff
@@ -305,6 +305,52 @@ export const deleteCredentialHandler = async ({ ctx, input }: DeleteCredentialOp
     }
   }
 
+  // Check if the user is a team owner and disable stripe for the team event type
```

Comment:
If the user does not have Stripe installed, it also will not appear on the event type apps page.